fluentd/configs.d/openshift/es-ops-copy-config.conf

```diff
@@ -17,11 +17,16 @@
       # recreate/reload connections
       reload_connections false
       reload_on_failure false
-      flush_interval "#{ENV['OPS_FLUSH_INTERVAL'] || ENV['ES_FLUSH_INTERVAL'] || '5s'}"
+      flush_interval "#{ENV['OPS_FLUSH_INTERVAL'] || ENV['ES_FLUSH_INTERVAL'] || '1s'}"
       max_retry_wait "#{ENV['OPS_RETRY_WAIT'] || ENV['ES_RETRY_WAIT'] || '300'}"
       disable_retry_limit true
       buffer_type file
       buffer_path '/var/lib/fluentd/buffer-es-ops-copy-config'
-      buffer_queue_limit "#{ENV['BUFFER_QUEUE_LIMIT'] || '1024' }"
-      buffer_chunk_limit "#{ENV['BUFFER_SIZE_LIMIT'] || '1m' }"
+      buffer_queue_limit "#{ENV['BUFFER_QUEUE_LIMIT'] || '64' }"
+      buffer_chunk_limit "#{ENV['BUFFER_SIZE_LIMIT'] || '8m' }"
+      # the systemd journald 0.0.8 input plugin will just throw away records if the buffer
```

Comment:
I think we can remove this comment entirely.

Reply:
@portante file a new issue - this PR is a cherry-pick of a cherry-pick of commits that have already been pushed

Reply:
I see this in master in two places which do not appear to be related to this:
 * https://github.com/openshift/origin-aggregated-logging/blob/b93e416f6749ab668ceb7b01395d7158e984a565/fluentd/configs.d/filter-pre-mux-client.conf#L13
 * https://github.com/openshift/origin-aggregated-logging/blob/b93e416f6749ab668ceb7b01395d7158e984a565/fluentd/configs.d/user/secure-forward.conf#L17

Not sure how a cherry pick could have pulled back those messages to here.